### PR TITLE
Drop Ubuntu Core

### PR DESCRIPTION
**Pull request by simondeziel**
This is related to https://github.com/lxc/distrobuilder/pull/623

### CODE REVIEW

**Comment by stgraber** [comment]
This is very wrong :)

We still do test LXD on Ubuntu Core systems. And most of the files being modified here are still very much required.

**Comment by stgraber** [comment]
`images/ubuntu-core.yaml` can go away. Everything else must stay.

**Comment by simondeziel** [comment]
@stgraber, here's take 2.

File: jenkins/jobs/lxd-test-snap.yaml [hunk 1]
```diff
@@ -77,11 +77,8 @@
 
     builders:
     - shell: |-
-        if [ "${{dist}}" = "ubuntu-core-16" ] || [ "${{dist}}" = "ubuntu-core-18" ] || [ "${{dist}}" = "ubuntu-core-20" ]; then
-        cd ~/lxc-ci
-        else
-        cd /lxc-ci
-        fi
+        cd ~/lxc-ci 2>/dev/null || cd /lxc-ci
+        pwd
```
Comment (stgraber) [inline jenkins/jobs/lxd-test-snap.yaml:81]: why the pwd call?

Comment (stgraber) [reply jenkins/jobs/lxd-test-snap.yaml:81]: And I'd prefer `cd /lxc-ci || cd ~/lxc-ci` personally as a random `lxc-ci` in the home dir should not be select over the standard /lxc-ci

Comment (simondeziel) [reply jenkins/jobs/lxd-test-snap.yaml]: `pwd` was to leave a visible trace of which dir was used, I dropped it.

**Pull request merged**